src/schema/schema-generator.ts

```diff
@@ -227,6 +227,7 @@ export abstract class SchemaGenerator {
                 );
                 fieldsMap[field.schemaName] = {
                   type: this.getGraphQLOutputType(field.name, field.getType(), field.typeOptions),
+                  complexity: field.complexity || 1,
```

Comment:
Should we explicitly put `1` here? Isn't it default behavior of `graphql-query-complexity`?